<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>因素分析</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/datav-data.png</normaloff>../images/datav-data.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionAnalysis"/>
    <addaction name="actionPearson"/>
    <addaction name="actionRegression"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionImport">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actionAnalysis">
   <property name="text">
    <string>灰色关联</string>
   </property>
  </action>
  <action name="actionPearson">
   <property name="text">
    <string>皮尔逊</string>
   </property>
  </action>
  <action name="actionRegression">
   <property name="text">
    <string>回归分析</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
